website/src/pages/documents/document.page.tsx

```diff
@@ -59,10 +60,12 @@ const AnnotatedDocumentPage = (props: { id: string }) => {
   return (
     <Layout>
       <Helmet title={doc?.title} />
-      <main className={css.annotatedDocument}>
-        <DocumentTitleHeader doc={doc} showDetails={true} />
-        <TabSet doc={doc} />
-      </main>
+      <FormProvider>
```

Comment:
This provider is really high up in the component hierarchy, I think it should live within the word details panel, since it isn't needed until that particular component.